src/context.ts

```diff
@@ -865,6 +865,21 @@ export class Context<U extends Deunionize<tg.Update> = tg.Update> {
     })
   }
 
+  persistentChatAction(action: Shorthand<'sendChatAction'>[0], callback: () => Promise<void>, extra?: tt.ExtraSendChatAction) {
```

Comment:
Perhaps you tried adding `intervalDuration` to `ExtraSendChatAction` and reverted it, so it no longer exists in `persistChatAction` extra as well. Make this

```TS
extra?: tt.ExtraSendChatAction & { intervalDuration?: number }
```